openedx/features/content_type_gating/field_override.py

```diff
@@ -31,24 +31,33 @@ def get(self, block, name, default):
         if not problem_eligible_for_content_gating:
             return default
 
-        # Read the group_access from the fallback field-data service
-        with disable_overrides():
-            original_group_access = block.group_access
+        # Option 1
```

Comment:
@cpennington -- this was the only other vaguely reasonable solution I came up with... what do you think?

Reply:
Yeah, I thought about doing that (although my way was going to be a bit clumsier, because I forgot about `self.fallback_field_data`).

I think maybe Option 1 is better by virtue of using only publicly-exposed methods (rather than reaching into `block._dirty_fields`).